pyspeckit/spectrum/writers/fits_writer.py

```diff
@@ -39,9 +39,17 @@ def write_data(self, filename=None, newsuffix='out', clobber=True,
 
         # Generate a WCS header from the X-array
         if self.Spectrum.xarr._make_header(tolerance=tolerance):
+            # print('iteratig over:', self.Spectrum.xarr.wcshead)
             for k,v in self.Spectrum.xarr.wcshead.iteritems():
+                # print('v is type: %s and has a value: %s' % (type(v), v))
                 if v is not None:
-                    header[k] = v
+                    try:
+                        header[k] = v
+                    except ValueError:
+                        try:
+                            header[k] = v.value
+                        except AttributeError:
+                            pass
```

Comment:
Why `pass` here?  If the header is not settable, I think we might want to raise an exception